config.rb

```diff
@@ -58,6 +58,11 @@
 #   end
 # end
 
+#Directories locations
+set :css_dir, 'css'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.